### PR TITLE
added LogSoftmax layer; modified import syntax

### PR DESCRIPTION
**Pull request by WNoxchi**
See [Gist](https://gist.github.com/WNoxchi/17ed83c3a0450cb4939b264d94c74054) for testing. 

`from .layers import *` --> `from fastai.layers import *`  can now import darknet normally via: `from fastai.models import darknet`.
The LogSoftmax layer works with the NLL loss criterion set by fastai's `conv_learner.ConvLearner` to produce Cross Entropy loss -- this is done automatically through fastai's `ConvnetBuilder` when loading pretrained models, but darknet isn't pretrained yet.

See [fastai thread](http://forums.fast.ai/t/project-implement-yolo-v3-backbone-and-preact-resnet/14613/50?u=borz) for how this started.

### CODE REVIEW

**Comment by jph00** [comment]
I'd rather we moved towards using the loss functions with integrated softmax, rather than having them in the models.
